contracts/MetadataViews.cdc

```diff
@@ -415,4 +415,96 @@ pub contract MetadataViews {
             self.socials = socials
         }
     }
+
+    // A view to represent a single field of metadata on an NFT.
+    //
+    // This is used to get traits of individual key/value pairs along with some contextualized data about the trait
+    pub struct Trait {
+        // The name of the trait. Like Background, Eyes, Hair, etc.
+        pub let traitType: String
```

Comment:
Not opposed, `traitType` was what I used purely because it's what OpenSea calls it, no strong opinions there and `name` would probably be more easily understood/intuitive